### ISSUE TITLE
[A11y][WCAG 1.4.11] Add background color to outline button

### ISSUE THREAD

**Issue by kpeterson-exygy** [civiform/civiform#10341]
**Describe the solution presented in this issue.**
The outline button style should have a background color to maintain color contrast across different background colors. Noticed this issue on the completed pre-screener card.

**Link to Figma mocks:**
[Example of use case
](https://www.figma.com/design/RmhVJCLgAKWGENJLWWAtNS/CiviForm--Components--Styles?node-id=7112-14428&t=CEn7AYyfpzhvauxq-1)

**Additional Context**
N/A

**Done when:**

- [ ] `Outline button` has a background color of `netural/white` (`#FFFFFF`)

![Image](https://github.com/user-attachments/assets/fd09616b-e63e-4a8f-989c-3ddad1af1f00)

**Comment by kpeterson-exygy** [comment]
@watermandrewc Wanted to raise this issue with you because it involves the theming work you've been doing. Am I correct that the contrast check for customized colors is only with a `#FFFFFF` background?

**Comment by watermandrewc** [comment]
Hi Kyle, thanks for bringing this up! Yes, we're only checking for contrast with `#FFFFFF` for customized theme colors. I missed this use case in manual testing, but as-is, this outline button on top of a colored background could result in text not having a high enough contrast ratio to meet accessibility standards. Updating the button background to white would fix this.

**Comment by tallulahkay** [comment]
@kpeterson-exygy @watermandrewc the unsubmitted version of the pre-screener has a dark blue button - should this one also be dark blue for consistency?

![Image](https://github.com/user-attachments/assets/47adf3b9-5a85-44e4-81c9-4e3cb3b0bbba)

**Comment by kpeterson-exygy** [comment]
@tallulahkay All of the other submitted cards use the outline button, so I'd say let's stick with that.
